test/unit/services/DmsRejectedErrorsSpec.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.services
+
+import services.DmsRejectedErrors
+import services.model.DmsRejectedError
+import unit.base.UnitSpec
+
+class DmsRejectedErrorsSpec extends UnitSpec {
+
+  import DmsRejectedErrors._
+
+  "Dms Rejected Errors" should {
```

Comment:
I'd maybe have a test to double check those multi line messages are read in properly

Reply:
Added test